### PR TITLE
feat(frontend): ConvertValue component

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

Desktop (1 row = 1 ConvertValue component):
Screenshot 2024-11-11 at 21 33 19

Mobile Desktop (1 row = 1 ConvertValue component):
Screenshot 2024-11-11 at 21 33 27

### CODE REVIEW

File: src/frontend/src/lib/components/convert/ConvertValue.svelte [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	export let ref: undefined | string = undefined;
+</script>
+
+<div class="mb-2 flex w-full justify-between last:mb-0 md:items-center">
+	<label for={ref} class="mr-1 text-sm sm:mr-2 sm:text-base"><slot name="label" /></label>
+
+	<div class="flex flex-col items-end sm:flex-row sm:items-center">
+		<span
+			class={`mb-1 text-sm font-bold sm:mb-0 sm:text-base ${$$slots['secondary-value'] ? 'mr-0 sm:mr-3' : ''}`}
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/convert/ConvertValue.svelte:10]: is `mr-0` necessary? isn't it default?

File: src/frontend/src/lib/components/convert/ConvertValue.svelte [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+			><slot name="main-value" /></span
+		>
+
+		<span class="text-sm text-aurometalsaurus"><slot name="secondary-value" /></span>
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/convert/ConvertValue.svelte:14]: I would add the same TODO you used in PR #3452 , just to keep track of the colors

(comment https://github.com/dfinity/oisy-wallet/pull/3452/files#r1838102292)

File: src/frontend/src/lib/components/convert/ConvertValue.svelte (hunk 1 above)
Comment (AntonioVentilii) [inline src/frontend/src/lib/components/convert/ConvertValue.svelte:10]: not a call to aciotn, but usually, in the other components, we put the $$slots outside:

```typescript
	let amount = true;
	$: amount = nonNullish($$slots.amount);
```
	
just to keep better track

don't know if useful

**Review by AntonioVentilii — APPROVED** [review]
LGTM, just a few comments

File: src/frontend/src/lib/components/convert/ConvertValue.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/convert/ConvertValue.svelte:10]: Nice, updated it.

File: src/frontend/src/lib/components/convert/ConvertValue.svelte (hunk 2 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/convert/ConvertValue.svelte:14]: Done!

File: src/frontend/src/lib/components/convert/ConvertValue.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/convert/ConvertValue.svelte:10]: Nice catch, removed

**Pull request merged**